packages/fxa-payments-server/src/components/PaymentConsentCheckbox/en.ftl

```diff
@@ -1,5 +1,5 @@
 ## Component - PaymentConsentCheckbox
 
-# Variables: $price (String) - The amount of the subscription being billed, including currency, e.g. $10.00. 
-# $productName (String) - The name of the subscribed product, e.g. Mozilla VPN.
-payment-confirm-with-legal-links = I authorize { -brand-name-mozilla }, maker of { -brand-name-firefox } products, to charge my payment method { $price } for { $productName }, according to <termsOfServiceLink>Terms of Service</termsOfServiceLink> and <privacyNoticeLink>Privacy Notice</privacyNoticeLink>, until I cancel my subscription.
+payment-confirm-with-legal-links-static = I authorize { -brand-name-mozilla }, maker of { -brand-name-firefox } products, to charge my payment method for the amount shown, according to <termsOfServiceLink>Terms of Service</termsOfServiceLink> and <privacyNoticeLink>Privacy Notice</privacyNoticeLink>, until I cancel my subscription.
```

Comment:
This seems to be reverting a change from a [recent PR](https://github.com/mozilla/fxa/pull/15092) by @IvoJP . Is this intentional?

Reply:
Hey @bcolsson yes it is. It actually caused a regression: FXA-7174. This ticket will also revert the copy change and go back to the original copy.

Please let me know if there are any concerns with this or if I have to update the id to a new one. I know it was removed entirely in the last PR, so I was hoping I could just add it back.

Reply:
Thanks, understood. In an ideal world we'd want to avoid having to revert strings (especially if we decide to try to update the string again at some point) as each locale ends up having to translate the same string multiple times.
For this, to avoid the above I'll need to run a migration from a previous change set as the translations have been removed from Pontoon at this point, but it's doable. In the future I would appreciate it if you called out a change that's reverting a string.

Reply:
Will do @bcolsson and thank you! I will alert you immediately next time that happens.